docs/manifests.md

```diff
@@ -1,28 +1,34 @@
 # Manifest Deployer
 
-To run k0s with your preferred extensions you have two options.
+Included with k0s, Manifest Deployer is one of two methods you can use to run k0s with your preferred extentions (the other being by [defining your extensions as Helm charts](helm-charts.md).
```

Comment:
```suggestion
Included with k0s, Manifest Deployer is one of two methods you can use to run k0s with your preferred extensions (the other being by [defining your extensions as Helm charts](helm-charts.md).
```

Reply:
(like before, both of the methods require "defining your extensions as helm charts")

Reply:
Got it.